packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -696,15 +697,23 @@ const verifyBuildScript = async (
 	const TEST_PORT = Math.ceil(Math.random() * 1000) + 20000;
 
 	const devProc = spawnWithLogging(
-		[npx, "wrangler", "pages", "dev", outputDir, "--port", `${TEST_PORT}`],
+		[
+			npx,
+			"wrangler@3.60.3",
+			"pages",
+			"dev",
+			outputDir,
+			"--port",
+			`${TEST_PORT}`,
+		],
 		{
 			cwd: projectPath,
 		},
 		logStream,
 	);
 
 	// Wait a few seconds for dev server to spin up
-	await sleep(7000);
+	await sleep(15000);
```

Comment:
Is it possible to ping the server instead of waiting for an arbitrary? It might take only a few seconds on some machines.

Reply:
yes definitely 🙂

I did had it that way earlier in this PR too

anyways I am just messing around and trying to understand why the CI e2es are failing so this is not really code I necessarily want to commit 😅